app/src/main/java/com/example/moviedb/ui/adapter/MovieAdapter.kt

```diff
@@ -32,6 +34,8 @@ class MovieAdapter(
     class MovieViewHolder(val binding: ItemMovieBinding) : ViewHolder(binding.root) {
         fun bindData(movie: Movie) {
```

Comment:
ở đây binding lặp lại hơi nhiều, nếu là em em sẽ dùng with(binding) =)), tùy người 
```suggestion
       fun bindData(movie: Movie) = with(binding) {
            textTitle.text = movie.name
            Glide.with(root).load(ImageUtils.getLink(movie.backdrop)).into(imagePoster)
            textVoteAverage.text = movie.voteAverage.toString()
        }
```